src/3.0/validate/validate.ts

```diff
@@ -78,7 +85,7 @@ const contextLoader = () => {
       const promise = nodeDocumentLoader(url);
       contexts.set(
         url,
-        promise.then(({ document }) => document)
+        promise.then(({ document }: { document: any }) => document)
```

Comment:
why ?

Reply:
i removed it, it was throwing a implicit error because of my code, but after the updated one liner it is not throwing any eslint error.